homeworks/rostyslav.havuka_RostyslavHavuka/6-typeScript/src/main.css

```diff
@@ -0,0 +1,54 @@
+.red-square {
+  background-color: rgb(255, 0, 0);
+  position: fixed;
+  right: 49%;
+  top: 33%;
+}
+
+input[data-key] {
```

Comment:
Create a separate class for the input. Fix this for all following classes that use [data] query